src/pages/Profile.jsx

```diff
@@ -33,14 +38,26 @@ const Profile = () => {
 
   // Documents related states
   const { session } = useSession();
-  const [showModal, setShowModal] = useState(false);
+  const [showAddDocModal, setShowAddDocModal] = useState(false);
+  const [showAclPermsDocContainerModal, setShowAclPermsDocContainerModal] = useState(false);
+  const [showAclPermissionModal, setShowAclPermissionModal] = useState(false);
+  const [documentName, setDocumentName] = useState('filename.jpg');
```

Comment:
I wouldn't put a false filename in here as a default. It seems like something that could easily lead to errors.